### DISCUSSION TITLE
Disable user.github.io page from displaying on profile

### DISCUSSION THREAD

**Discussion by anathonous** [community/community#23642] [New to GitHub]
<p>How can I prevent a pages <a href="http://username.github.io" rel="noopener nofollow ugc">username.github.io</a> repository from displaying on my github profile?</p>
<p>Thank you,<br>
-Nate</p>

**Comment by otter-computer** [comment, marked as answer]
<p>Hi <a class="mention" href="/blackvslight">@blackvslight</a>, welcome to the GitHub Support Community!</p>
<p>Do you mean how do you stop it appearing at the top of your profile here?</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/blackvslight" target="_blank" rel="noopener">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/2439407/181078611-ecd53deb-0ac4-447b-bf59-9163567ee753.jpeg" class="thumbnail onebox-avatar" width="400" height="400">

<h3><a href="https://github.com/blackvslight" target="_blank" rel="noopener">blackvslight - Overview</a></h3>

<p>blackvslight has one repository available. Follow their code on GitHub.</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>
<br>
<div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/6/8/687973cc322096168fe912ef14d199cd5e680f3e.jpeg" data-download-href="/uploads/short-url/eUdRM6ACtaMiAiV2GWUyq1zyuya.jpeg?dl=1" title="Screenshot 2020-11-12 at 15.55.40"><img src="https://user-images.githubusercontent.com/2439407/181078615-400a210d-002f-4483-aeb4-0037b1855688.jpeg" alt="Screenshot 2020-11-12 at 15.55.40" data-base62-sha1="eUdRM6ACtaMiAiV2GWUyq1zyuya" width="690" height="270" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/6/8/687973cc322096168fe912ef14d199cd5e680f3e_2_690x270.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/6/8/687973cc322096168fe912ef14d199cd5e680f3e_2_1035x405.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/6/8/687973cc322096168fe912ef14d199cd5e680f3e_2_1380x540.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/6/8/687973cc322096168fe912ef14d199cd5e680f3e_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot 2020-11-12 at 15.55.40</span><span class="informations">1976×776 177 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>This area of your profile shows either popular repositories that you own or repositories you pin to your profile. There isn’t a way to remove this section or hide a repository from here directly. The only way to remove this specific repository would be to pin another repository that you own to this section of your profile.</p>

**Comment by anathonous** [comment]
<p>Thanks, Solved. Cheers duder.</p>
